pyhealth/datasets/mimic3.py

```diff
@@ -101,9 +102,10 @@ def parse_basic_info(self, patients: Dict[str, Patient]) -> Dict[str, Patient]:
         # sort by admission and discharge time
         df = df.sort_values(["SUBJECT_ID", "ADMITTIME", "DISCHTIME"], ascending=True)
         # group by patient
-        df_group = df.groupby("SUBJECT_ID")
-        # load patients
-        for p_id, p_info in tqdm(df_group, desc="Parsing PATIENTS and ADMISSIONS"):
+        df_group = df.groupby("SUBJECT_ID", group_keys="SUBJECT_ID")
```

Comment:
What does this `group_keys` do? Per [here](http://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.groupby.html) it seems like `group_keys` should be a boolean